chapters/CRUD-Operations.md

```diff
@@ -5,83 +5,200 @@ for the **did:btcr2** method.
 
 ### Create
 
-The Create operation consists of two main algorithms for creating identifiers
-and DID documents. A **did:btcr2** identifier and DID document can either be created
-from a deterministic key pair or from an external ::Intermediate DID Document::.
-In both cases, DID creation can be undertaken in an offline manner, i.e., the DID
-controller does not need to interact with the Bitcoin network to create their DID.
+Creating a **did:btcr2** identifier is entirely offline, requiring no innate
+network interactions to generate a new identifier. Each creation starts either
+with a public key or a ::Genesis Document::.  Both creation algorithms first
+create the ::Genesis Bytes:: that commit to an ::Initial DID Document::.
 
-#### From Deterministic Key Pair
+To create a **did:btcr2** identifier from a public key without an Initial DID
+Document, use [Algo 1: Create Genesis Bytes from Public Key], then encode those
+bytes as in [Algo 3: Encode Identifier].
 
-The From Deterministic Key Pair algorithm encodes a secp256k1 public key
-as a **did:btcr2** identifier. The public key is then used to deterministically
-generate the ::initial DID document::.
+To create a **did:btcr2** identifier from a ::Genesis Document::, use
+[Algo 2: Create Genesis Bytes from Genesis Document], then encode those
+bytes along with a version, network for the identifier and an identifier type of
+“external” using [Algo 3: Encode Identifier].
 
-It takes the following inputs:
+The output of encoding the identifier is the newly created DID.
 
-* `pubKeyBytes` - a compressed SEC encoded secp256k1 public key; REQUIRED; bytes
-* `version` - the identifier version; OPTIONAL; integer; default=1.
-* `network` - the Bitcoin network used for the identifier; OPTIONAL; string; default=`"bitcoin"`.
+Note: When creating from a ::Genesis Document::, it is likely that creators
+will want to include information, such as ::Beacons:: and other service
+endpoints, which requires online interactions, e.g., to establish a unique
```

Comment:
Note needs to help help clarify that these DIDs still require no onchain interactions.

"Requires network interactivity but does not require onchain anchoring"

We should use this language throughout the spec. Not online interactions.